build_master_index

```diff
@@ -3,95 +3,79 @@
 #PBS -N build_cc_db
 #PBS -P v45
 #PBS -l storage=gdata/ik11+gdata/cj50+gdata/hh5+gdata/qv56
-#PBS -l walltime=6:00:00 
-#PBS -l mem=64GB 
+#PBS -l walltime=8:00:00
+#PBS -l mem=48GB
 #PBS -l ncpus=1
 #PBS -q normalbw
 #PBS -l wd
 #PBS -W block=true
 #PBS -j oe
-#PBS -o logs
-#PBS -e logs
+#PBS -e log.err
+#PBS -o log.out
```

Comment:
You're merging the error and output streams and then specifying separate files for those. Is that what you intended?

Reply:
Ups, I missed the `-j `option there. Can I safely delete`#PBS -e log.err`?

Reply:
Yes I think so. Just wasn't sure which way you wanted to go.